### ISSUE TITLE
Error in laplace_KL_divergence (cross-entropy) calculation

### ISSUE THREAD

**Issue by joefowler** [usnistgov/mass#115]
**[Original report](https://bitbucket.org/joe_fowler/mass/issues/115) by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

The following function:
```
def problem(s=100):
    np.random.seed(s)
    x = mass.MnKAlphaDistribution().rvs(size=1000)
    gain = np.linspace(-.01, .01, 21)
    for p in gain:
        xg = np.exp(p)*x
        D = mass.mathstat.entropy.laplace_KL_divergence(xg, x, w=3.0, approx_method="exact")
        print p, D
```

yields:

```text
-0.01 5.03225233547
-0.009 5.06380581439
-0.008 4.71862087745
-0.007 4.50721121518
-0.006 4.23901758902
-0.005 3.91742505395
-0.004 3.16393655175
-0.003 2.14315454976
-0.002 1.08519445413
-0.001 0.37497307604
0.0 -0.0172018059003
0.001 0.399868073069
0.002 1.22083344994
0.003 3.74144419157e+50  ### <-- PROBLEM!
0.004 2.96397528679
0.005 3.99156603325
0.006 5.1943620973
0.007 6.10309518463
0.008 5.16449975617e+120   ### <-- PROBLEM!
0.009 5.8867039354
0.01 5.85026087114
```

The function values (2nd column) should be a smooth-ish function, but there's a bug yielding many order of magnitude error in a couple of cases.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Fix an underflow error in entropy computations.

Fixes bug #115.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Removing milestone: v0.6 (automated comment)
